src/content/app/tools/blast/components/blast-submission-header/BlastSubmissionHeader.scss

```diff
@@ -73,3 +69,35 @@
 .controlButtons > * {
   flex-shrink: 0;
 }
+
+.deleteMessageContainer {
+  grid-area: control-buttons;
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+  column-gap: 30px;
+}
+
+.deleteMessage {
+  color: $red;
+}
+
+.clickable {
+  color: $blue;
+  cursor: pointer;
+}
+
+// Leave media queries at the bottom of the file to make sure they override CSS rules defined above
+@media screen and (max-width: 1720px) {
```

Comment:
@ens-ma7 Are we going ahead with this view for screen size 1720px? 
![Screenshot 2022-09-13 at 10 53 11](https://user-images.githubusercontent.com/6347854/189871316-ceaa1b89-4e03-42d4-8fa0-e64d550bcb82.png)

Reply:
@jyothishnt i will probably revert this change if we are going with the show/hide option for showing the deletion message